application/models/FailedLoginAttempt.php

```diff
@@ -91,17 +91,20 @@ public function isLockedOut(string $attemptType): bool
 
         switch ($attemptType) {
             case FailedLoginAttempt::TYPE_LOGIN:
-                $timeOut = Yii::app()->getConfig('timeOutTime');
-                $maxLoginAttempt = Yii::app()->getConfig('maxLoginAttempt');
+                $timeOut = intval(App()->getConfig('timeOutTime'));
+                $maxLoginAttempt = intval(App()->getConfig('maxLoginAttempt'));
                 break;
             case FailedLoginAttempt::TYPE_TOKEN:
-                $timeOut = Yii::app()->getConfig('timeOutParticipants');
-                $maxLoginAttempt = Yii::app()->getConfig('maxLoginAttemptParticipants');
+                $timeOut = intval(App()->getConfig('timeOutParticipants'));
+                $maxLoginAttempt = intval(App()->getConfig('maxLoginAttemptParticipants'));
                 break;
             default:
                 throw new InvalidArgumentException(sprintf("Invalid attempt type: %s", $attemptType));
         }
-
+        // Return false if disable my maxLoginAttempt
+        if ($maxLoginAttempt <= 0) {
```

Comment:
Are you sure about this? This seems like new behaviour.
Can this, by accident, make this feature to be not active?
Maybe set a default fallback value of 5? (Yes, arbitrary)

Reply:
>  Can this, by accident, make this feature to be not active?

If someone delete before the value here : he clearly want to disable it.

>  Maybe set a default fallback value of 5? (Yes, arbitrary)

No, never : 
1. I really don't want to disable 1 : it's the **best** solution for token (with 1 second)
2. allow user to disable BUT register the save the number of attempts can be really interesting.